### ISSUE TITLE
Does not work when used with variables and hsl

### ISSUE THREAD

**Issue by steverandy** [jonathantneal/postcss-color-mod-function#24]
Repo for demo: https://github.com/steverandy/postcss-demo/tree/reduced

Actual output:
```css
:root {
  --rgb: #b366ff;
  --hsl: hsl(270, 100%, 70%);
}

test {
  --test-rgb: rgba(179, 102, 255, 0.25);
  --test-hsl: hsla(270, 100%, 70%, 0.25);
  --test-rgb-var: rgba(179, 102, 255, 0.25);
  --test-hsl-var: color-mod(hsl(270, 100%, 70%) alpha(25%));
}
```

Expected output:
```css
:root {
  --rgb: #b366ff;
  --hsl: hsl(270, 100%, 70%);
}

test {
  --test-rgb: rgba(179, 102, 255, 0.25);
  --test-hsl: hsla(270, 100%, 70%, 0.25);
  --test-rgb-var: rgba(179, 102, 255, 0.25);
  --test-hsl-var: hsla(270, 100%, 70%, 0.25);
}
```

**Comment by jonathantneal** [comment]
@steverandy, thank you so much for being diligent with this bug. The issue has been resolved in 3.0.3! 🎉

**Comment by steverandy** [comment]
Thank you for pushing the fix so quickly!
